### PR TITLE
Patch 2

### LINKED ISSUE #30685: Typo in Documentation at docs/samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs in multiple lines
There are 3 Typos in the [documentation ](https://github.com/dotnet/docs/blob/main/samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs#:~:text=_decryptOpeFileDialog.InitialDirectory%20%3D%20EncrFolder%3B) at line 95, 96 and 98. There is a missing "n" in _decryptOpe**n**FileDialog 
image

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2a8e441d-cd59-de15-a814-0d86d4cda052
* Version Independent ID: c94ca6d4-7e93-8aac-93d1-7f249c77d243
* Content: [Walkthrough: Create a Cryptographic Application](https://docs.microsoft.com/en-us/dotnet/standard/security/walkthrough-creating-a-cryptographic-application)
* Content Source: [docs/standard/security/walkthrough-creating-a-cryptographic-application.md](https://github.com/dotnet/docs/blob/main/docs/standard/security/walkthrough-creating-a-cryptographic-application.md)
* Product: **dotnet**
* Technology: **dotnet-security**
* GitHub Login: @IEvangelist
* Microsoft Alias: **dapine**

### PR DESCRIPTION
**Pull request by mudassir014**
## Summary

Fixed the typo in both files

Fixes #30685 (if available)

### CODE REVIEW

**Comment by mudassir014** [comment]
Hi @BillWagner 
I created a small PR with one failed check. I am not able to figure out the issue. I would be thankful if you could help me out here as I am pretty sure it won't take much of your time.
Thanks :)

File: samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.Designer.cs [hunk 1]
```diff
@@ -102,8 +102,8 @@ private void InitializeComponent()
             //
             // openFileDialog2
             //
-            this._decryptOpeFileDialog.Filter = "Encrypted files (*.enc) | *.enc|All files (*.*)|*.*";
-            this._decryptOpeFileDialog.InitialDirectory = "c:\\encrypt";
+            this._decryptopenFileDialog.Filter = "Encrypted files (*.enc) | *.enc|All files (*.*)|*.*";
+            this._decryptopenFileDialog.InitialDirectory = "c:\\encrypt";
```
Comment (IEvangelist) [inline samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.Designer.cs:106]:
```suggestion
            this._decryptOpenFileDialog.Filter = "Encrypted files (*.enc) | *.enc|All files (*.*)|*.*";
            this._decryptOpenFileDialog.InitialDirectory = "c:\\encrypt";
```

File: samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.Designer.cs [hunk 2]
```diff
@@ -183,7 +183,7 @@ private void InitializeComponent()
         private System.Windows.Forms.Button buttonEncryptFile;
         private System.Windows.Forms.OpenFileDialog _encryptOpenFileDialog;
         private System.Windows.Forms.Button buttonCreateAsmKeys;
-        private System.Windows.Forms.OpenFileDialog _decryptOpeFileDialog;
+        private System.Windows.Forms.OpenFileDialog _decryptopenFileDialog;
```
Comment (IEvangelist) [inline samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.Designer.cs:186]:
```suggestion
        private System.Windows.Forms.OpenFileDialog _decryptOpenFileDialog;
```

**Review by IEvangelist — COMMENTED** [review]
Typo in casing

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Comment by BillWagner** [comment]
Hi :wave: @mudassir014 

This sample requires Visual Studio for the build. To fix the build error, you need to add a `snippets5000.json` file to this folder that specifies building with VS.  You can find an example [here](https://github.com/dotnet/docs/blob/main/samples/snippets/csharp/roslyn-sdk/SemanticQuickStart/snippets.5000.json)

**Pull request closed without merging**